internal/cli/cmd/version.go

```diff
@@ -48,16 +68,26 @@ func getLatestStableVersion() string {
 }
 
 func newCmdVersion() *cobra.Command {
-	return &cobra.Command{
+	cmd := &cobra.Command{
 		Use:   "version",
 		Short: "Display detailed version information",
 		Long:  `Displays information about the version of this software.`,
-		Run: func(cmd *cobra.Command, _ []string) {
-			// TODO: add support for reporting the Cilium version running in
-			// the cluster, if any. See https://github.com/cilium/cilium-cli/issues/131
+		RunE: func(cmd *cobra.Command, args []string) error {
 			fmt.Printf("cilium-cli: %s compiled with %v on %v/%v\n", Version, runtime.Version(), runtime.GOOS, runtime.GOARCH)
 			fmt.Printf("cilium image (default): %s\n", defaults.Version)
 			fmt.Printf("cilium image (stable): %s\n", getLatestStableVersion())
+			ciliumVersion := NewCiliumVersion()
+			version, err := ciliumVersion.GetRunningCiliumVersionForCluster(context.Background())
+			if err != nil {
+				fmt.Printf("cilium image (running): unknown. Error: %v \n", err)
```

Comment:
Nit: drop the trailing space after the error message

```suggestion
				fmt.Printf("cilium image (running): unknown. Error: %v\n", err)
```